lib/actions/apiV2.js

```diff
@@ -436,11 +438,15 @@ export const fetchNearby = (position, radius) => {
                 lon
                 code
                 gtfsId
+                stopRoutes: routes (serviceDates: $currentServiceWeek) {
```

Comment:
why are we renaming routes to stop routes?

Reply:
for OTP1 support!